src/fixes/quote_character.rs

```diff
@@ -0,0 +1,94 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct QuoteCharacterFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for QuoteCharacterFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "QuoteCharacter",
+        }
+    }
+}
+
+impl Fix for QuoteCharacterFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_line(&self, line: &mut LineEntry) -> Option<()> {
+        let value = line.get_value()?;
+        let stripped = value.strip_prefix("\"").unwrap_or(&*value);
```

Comment:
In addition to the double-quotes (`"`), you should remove the single-quotes (`'`).

Reply:
Thank you! Missed that, will fix it.